README.md

```diff
@@ -33,620 +33,677 @@ If you want to build the CLI using the source code, please, take into account th
-53f0f126d8a5975a1c000429   Magento           {"memory":1024}            53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c00042a   MongoDB           {}                         53f0f0e3d8a5975a1c000070
-53f0f126d8a5975a1c00042b   Wordpress         {"memory":2048}            53f0f0e3d8a5975a1c000070
-5409b197d8a597d977000010   Docker            {}                         5409b147d8a597d977000001
-```
-You can choose whatever application/stack is fine for your purpose, we choose Wordpress. Take note of the application identifier, `53f0f126d8a5975a1c00042b` for Wordpress.
-We will also need the location where we want our server to be instantiated. Execute concerto wizard locations list to get the possible locations and its identifier.
+
+Let's type `concerto wizard apps list` to check what servers can I instantiate using IMCO CLI wizard.
+
+```bash
+$ concerto wizard apps list
+ID                         NAME              FLAVOUR_REQUIREMENTS   GENERIC_IMAGE_ID
+5aabb75a1de0240abb000185   Ubuntu 14.04      {}                     5aabb7551de0240abb000064
+5aabb75a1de0240abb000186   Ubuntu 16.04      {}                     5aabb7551de0240abb000065
+5aabb75a1de0240abb000187   Windows 2012 R2   {"memory":4096}        5aabb7551de0240abb000066
+5aabb75a1de0240abb000188   Joomla            {"memory":1024}        5aabb7551de0240abb000064
+5aabb75a1de0240abb000189   Magento           {"memory":1024}        5aabb7551de0240abb000064
+5aabb75b1de0240abb00018a   MongoDB           {}                     5aabb7551de0240abb000064
+5aabb75b1de0240abb00018b   Wordpress         {"memory":1024}        5aabb7551de0240abb000064
+5aabb75b1de0240abb00018c   Docker            {"memory":2048}        5aabb7551de0240abb000064
 ```
+
+You can choose whatever application/stack is fine for your purpose, we choose `Wordpress`. Take note of the application identifier, `5aabb75b1de0240abb00018b` for `Wordpress`.
+
+We will also need the location where we want our server to be instantiated. Execute `concerto wizard locations list` to get the possible locations and its identifier.
+
+```bash
 $ concerto wizard locations list
-ID                          NAME
-53f0f0e2d8a5975a1c00006b    USA
-53f0f0e2d8a5975a1c00006c    Europe
-53f0f0e2d8a5975a1c00006d    Asia Pacific
-53f0f0e2d8a5975a1c00006e    South America
+ID                         NAME
+5aabb7551de0240abb000060   North America
+5aabb7551de0240abb000061   Europe
+5aabb7551de0240abb000062   Asia Pacific
+5aabb7551de0240abb000063   South America
+```
+
+Take note of your preferred location. We will use `5aabb7551de0240abb000060` for `North America`.
+
+When using IMCO's Web UI, the wizard takes care of filtering appropriate cloud accounts for that provider and location. However, using the CLI is the user's responsibility to chose a provider cloud account for that application/stack and location; and a server plan capable of instantiating the stack in that location.
+To show all possible cloud accounts execute this command:
```

Comment:
The command below shows cloud providers, not cloud accounts.
Also, we should stop showing the required credentials, since they are no longer of use in the CLI.